### ISSUE TITLE
clarify how to get credentials

### ISSUE THREAD

**Issue by judell** [turbot/steampipe-plugin-zoom#8]
We say:

```
# Zoom API credentials are available to users with Developer role in
# the account
```

I've never had my own Zoom account, just signed up, wasn't 100% clear how to get the key. 

1. Go to https://marketplace.zoom.us/develop/create
2. Click JWT and click Create
3. Fill in basic info: name, email
4. Go to App Credentials and you will find the API Key and Secret

**Comment by rajlearner17** [comment]
@judell thanks for raising this, we will make this small change to have more clarity.

**Comment by cbruno10** [comment]
Hey @judell , we have some instructions linked in https://hub.steampipe.io/plugins/turbot/zoom#credentials in the `Credentials` row, that ultimately links to https://marketplace.zoom.us/docs/guides/build/sdk-app. I believe this page has roughly the same steps that you outlined above (though annoyingly, the `Create a JWT App` link toward the top of the page is broken, but the actual steps are below it anyway).

Did you find this link within the docs and then find it hard to follow, or was it difficult to find that link on our Hub page?

**Comment by judell** [comment]
Agree that linking to https://marketplace.zoom.us/docs/guides/build/sdk-app is the right answer.
